### ISSUE TITLE
Buildpack: Incompatible Microsoft.AspNetCore.App version 

### ISSUE THREAD

**Issue by mshearer0** [wietsevenema/hello-docker-go#1]
When trying to run the buildpack image (pg50 of book) i get:

It was not possible to find any compatible framework version
The framework 'Microsoft.AspNetCore.App', version '2.2.0' was not found.
...

I can't find where this specific dependency is referenced. 

I'm using Google CloudShell.

**Comment by wietsevenema** [comment]
Thank you for letting me know, I'll try to reproduce this

**Comment by wietsevenema** [comment]
This is what I did to try to reproduce it: 

1. I opened a Google Cloud Shell, cloned this repository. 
`git clone https://github.com/wietsevenema/hello-docker-go.git`
2. Changed to the directory with the clone
`cd hello-docker-go`
3. Ran pack to build the image
`pack build --builder gcr.io/buildpacks/builder:v1 hello-buildpack`

That worked. The only thing I can think of is that you're in a different directory when you run `pack build`? That does needs the source.
